jenkins/scripts/integration_test.sh

```diff
@@ -53,13 +53,9 @@ KUBERNETES_VERSION_UPGRADE_FROM="${KUBERNETES_VERSION_UPGRADE_FROM:-}"
 KUBERNETES_VERSION_UPGRADE_TO="${KUBERNETES_VERSION_UPGRADE_TO:-}"
 KUBECTL_SHA256="${KUBECTL_SHA256:-}"
 
-if [[ "${IRONIC_INSTALL_TYPE}" == "source" ]]; then
+if [[ "${IRONIC_INSTALL_TYPE}" == "source" ]] || [[ "${REPO_NAME}" == "ironic-image" ]]; then
```

Comment:
```suggestion
if [[ "${REPO_NAME}" == "ironic-image" ]]; then
```
No need to check for the install type, we'll drop the RPM support completely.

Also,
/retitle 🌱 Build ironic always from source